File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/ImagesResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class ImagesResponse {
    public Images Images { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ImagesResponse {\n");
      sb.Append("  Images: ").Append(Images).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Android/Aspose-PDF-Cloud-SDK-for-Android/src/main/java/com/aspose/pdf/model/ImagesResponse.java
package com.aspose.pdf.model;

import com.aspose.pdf.model.Images;
public class ImagesResponse {
  private Images Images = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getImages
	 * Gets Images
	 * @return Images
	 */
  public Images getImages() {
    return Images;
  }

	/**
	 * setImages
	 * Sets Images
	 * @param Images Images
	 */
  public void setImages(Images Images) {
    this.Images = Images;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ImagesResponse {\n");
    sb.append("  Images: ").append(Images).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

